src/APIFunctions/3DPrinting.js

```diff
@@ -50,14 +60,14 @@ export async function submit3DPrintRequest (printRequest) {
  * @returns {(Object[]|string)} - The list of requests or error from the API.
  */
 export async function getAll3DPrintRequests () {
-  let allRequests = []
+  const allRequests = new ApiResponse()
   await axios
     .post('/api/3DPrintingForm/GetForm', {})
     .then(result => {
-      allRequests = result.data
+      allRequests.responseData = result.data
     })
     .catch(err => {
-      allRequests = err
+      allRequests.error = err
```

Comment:
remember, the error is a boolean so set it to true or false
you also would want to set responseData to the error like

```js
allRequests.error = true
allRequests.responseData = err
```